### PR TITLE
Show user, Statistics in Admin #2550, Skill Example #2635

### LINKED ISSUE #2550: Admin Dashboard: Show User and Skill Statistics
The information on user activities is limited on the admin dashboard.

Please implement a chat for user information with the following:
* Logins over time
* User registration over time
* Skill edits over time
* Skill creations over time
* Any other interesting information

Screenshot from 2019-07-14 18-11-28

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2550, #2635

Changes:
- [x] Fix show more example
- [x] Skill edits over time
- [x] Skill creations over time
- [x] Skill Uses over Time
- [x] Logins over time
- [x] User registration over time
- [x] Created reusable line chart component

Demo Link: https://pr-2666-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Screenshot 2019-07-25 at 2 37 19 AM
Screenshot 2019-07-25 at 2 37 27 AM

Screenshot 2019-07-24 at 10 18 36 PM

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd I have made the changes, created a new PR as earlier PR was not linked to my GitHub account.

Screenshot 2019-07-25 at 3 07 03 AM

**Comment by fragm3** [comment]
@harsh-jindal It was correct, was from older PR which I closed, anyways changed it

File: src/components/Admin/AdminTab/AdminTab.js [hunk 1]
```diff
@@ -111,6 +139,77 @@ class AdminTab extends React.Component {
...
+
+    const CONFIG = [
+      {
+        data: lastModifiedOverTime,
+        heading: 'Skill Edits over Time',
+        legend: 'Skill edits',
+      },
+      {
+        data: lastAccessOverTime,
+        heading: 'Skill Uses over Time',
```
Comment (akshatnitd) [inline src/components/Admin/AdminTab/AdminTab.js]: Skill Usage

**Comment by harsh-jindal** [comment]
@fragm3 Set the width of the skill edits over time in laptop view as 1240px. Currently it looks a little odd. Setting to 1240px means covering the exact width covered by the cards in first row (400 +20+400+20+400)
Screenshot 2019-07-25 at 5 13 55 PM

**Comment by fragm3** [comment]
@akshatnitd @harsh-jindal made the changes:

Screenshot 2019-07-25 at 9 30 40 PM

File: src/components/Admin/AdminTab/AdminTab.js (hunk 1 above)
Comment (akshatnitd) [reply src/components/Admin/AdminTab/AdminTab.js]: Change not made.

**Comment by fragm3** [comment]
Made the changes,
Screenshot 2019-07-26 at 12 18 32 AM
Screenshot 2019-07-26 at 12 18 42 AM

**Comment by fragm3** [comment]
@akshatnitd I tried looking into it, not sure why this is not working on server side

**Pull request merged**